symbolic-debuginfo/src/macho.rs

```diff
@@ -206,6 +257,7 @@ impl<'d> MachObject<'d> {
             symbols: self.macho.symbols(),
             sections,
             vmaddr: self.load_address(),
+            symbolmap: self.bcsymbolmap.clone(),
```

Comment:
I can’t remember if you mentioned this to me in the walkthrough, not sure if its worth avoiding the clone here?

Reply:
it would be somewhat hard to do effectively.  And in practice users (and symbolicator too) create a DebugSession which for MachO will always construct the entire symbol table anyway so there's no savings as this entire vector would be allocated during that operation anyway.

Reply:
`Arc` it? That's what we do in PDB's case.

Reply:
that is an excellent idea.  so obvious it should have been done from the start :)